static/src/js/components/GranuleResults/GranuleResultsDownloadNotebookButton.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useRef } from 'react'
+import ReactDOM from 'react-dom'
+import { Dropdown, Tab } from 'react-bootstrap'
+import { PropTypes } from 'prop-types'
+import { Download, CloudFill } from '@edsc/earthdata-react-icons/horizon-design-system/hds/ui'
+import { FaFileCode } from 'react-icons/fa'
+
+import Button from '../Button/Button'
+import CopyableText from '../CopyableText/CopyableText'
+import EDSCTabs from '../EDSCTabs/EDSCTabs'
+import ExternalLink from '../ExternalLink/ExternalLink'
+
+import { addToast } from '../../util/addToast'
```

Comment:
I don't see addToast or getFilenameFromPath being used anywhere. Just wanted to point that out since eslint wasn't.